src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -7,14 +7,27 @@ $(function () {
     addFacility_handler();
     //handle delete facility
     deleteFacility_handler();
+    //handle change in filters in facility
+    facilityFilter_handler();
 });
 
 function getFacilitiesInfo() {
+ 
     let facilityIds = [];
     ipcRenderer.send('Facility:getFacilitiesByRegion', parseInt(window.process.argv.slice(-1)));
     ipcRenderer.once('Facility:getFacilitiesByRegionOK', (e, res) => {
         dataAvailable = true;
         if (res != null) {
+            $("#facilityList").empty();
+            let functionalFacilities = $('input[name=functionalFacilitiesDisplay]:checked').val();  
+
+        
+            if (functionalFacilities === 'functional') {
+                res = res.filter(facility=>facility.isFunctional)
+            } else if (functionalFacilities === 'non-functional') {
+                res = res.filter(facility=>!facility.isFunctional)
+            }
```

Comment:
Replace all magic strings here with constants. Avoid magic strings and numbers

Reply:
Are these HTML?

Reply:
What magic string?

Reply:
Magic strings are strings directly in code. There's a lot of it here. Extract them into constant variables if possible, but I'm asking if this is HTML?

Reply:
From discussion on Discord: `functional` and `non-functional` are not HTML so they should be extracted. I suggest an `enum` (do we have that in JS?)

Reply:
I think the typical thing to do is store them as constants like const FUNCTIONAL = 'functional', we can do an enum file but thats way more complex

Reply:
Can't we put the enum in the same file?

Reply:
We do have enum in JS, but we also need to put the enum in the HTML.

Reply:
Hm, then just `const` is fine